packages/source-iottwinmaker/src/property-value/client/getPropertyValueByEntity.ts

```diff
@@ -0,0 +1,60 @@
+import { GetPropertyValueCommand, GetPropertyValueResponse, IoTTwinMakerClient } from '@aws-sdk/client-iottwinmaker';
+import { DataType, DataStreamBase } from '@iot-app-kit/core';
+import { TwinMakerStaticDataQuery } from '../types';
+import { toValue } from '../../utils/propertyValueUtils';
+import { isNumber } from 'lodash';
+
+export const getPropertyValueByEntity = async ({
+  client,
+  query,
+}: {
+  client: IoTTwinMakerClient;
+  query: TwinMakerStaticDataQuery;
+}): Promise<DataStreamBase[]> => {
+  const results: DataStreamBase[] = [];
+  let nextToken: string | undefined = undefined;
+  do {
+    const response: GetPropertyValueResponse = await client.send(
```

Comment:
Looks like return type of this GetPropertyValueResponse and gets the property values for a component, component type, entity, or workspace. But not about the history of a time series property value? Are you handling that separately?

Reply:
We should also test with large response data to test the performance of the fetching propertyValue and calculate the average time to parse and fetch results. That depends on p90/p99 of API.

Reply:
I have a separate task to test with a large amount of data bindings